public/js/dec-bin-hex-converter

```diff
@@ -0,0 +1,47 @@
+(function(){
+
+    var ConvertBase = function (num) {
+        return {
+            from : function (baseFrom) {
+                return {
+                    to : function (baseTo) {
+                        return parseInt(num, baseFrom).toString(baseTo);
+                    }
+                };
+            }
+        };
+    };
+        
+    // binary to decimal
+    ConvertBase.bin2dec = function (num) {
```

Comment:
I think you should use a prototype for the inheritance that could happen later

ConvertBase.prototype.bin2dec